.gitmodules

```diff
@@ -2,3 +2,8 @@
 	path = tools/vcpkg
 	url = https://github.com/Microsoft/vcpkg
 	branch = master
+
+[submodule "third_party/ms-gsl"]
+	path = "third_party/ms-gsl"
+	url = https://github.com/microsoft/GSL
+	branch = master
```

Comment:
Right now it's pulling the latest. But since we use submodules - for CMake build we pin-point to specific commit ID. Technically the MS-GSL is only needed for older compilers without C++20 support, i.e. latest-latest Mac and latest gcc-9+ (as well as vs2019 with c++_latest flag) no longer require MS-GSL. As an alternative - I also want to add an option to build with ABSEIL instead of MS-GSL....

Re. pinning in Bazel - I am not sure how to do it. I wonder if we add Abseil, then test the combo of Bazel+Abseil, and CMake+GSL, that'd be covering the most needed build configurations for those who want to build statically. I can add CMake+Abseil build flavor.